pkg/config/config_test.go

```diff
@@ -0,0 +1,39 @@
+package config
+
+import (
+	"os"
+	"testing"
+
+	"github.com/moov-io/base/database"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConfig(t *testing.T) {
+	t.Run("When DATABASE_TYPE is not set", func(t *testing.T) {
+		os.Setenv("DATABASE_TYPE", "")
+
+		conf := New()
+		conf.Load()
+
+		require.Equal(t, conf.Database.SqlLite.Path, "customers.db")
+	})
+
+	t.Run("When DATABASE_TYPE is set to mysql", func(t *testing.T) {
+		os.Setenv("DATABASE_TYPE", "mysql")
```

Comment:
I need to restore env vars between tests...

Reply:
Setting env variables in tests usually doesn't work as expected. The value impacts all concurrently running test so you can easily run into problems as the tests are ran in different orderings each time.

Reply:
It seems that if test is not "marked as Parallel" it will be run in goroutine but sequential. Interesting [about t.Parallel](https://golang.org/pkg/testing/#T.Parallel):
> Parallel signals that this test is to be run in parallel with (and only with) other parallel tests.

So if we don't make this test parallel it's safe to work with ENV. Looking on how other test such cases I've found good helper to restore env after tests: https://github.com/spf13/viper/blob/f67a901790130504299091703639a69370244dd4/internal/testutil/env.go

Sounds good?

Reply:
Running tests in sequential order means they'll run a lot slower. The point of something like https://github.com/moov-io/base/pull/101 is to let tests run concurrently to each other.

Reply:
I don't know how we can ensure the ENV is safe over the long term. There's nothing in place to make sure we fail CI if a `t.Parllel()` slips in. Assuming tests will be ran in sequential order seems like something we'll forget in a few months.

Reply:
While in general I agree about the issues with parallel tests in the future, most probably this code will be removed in the next few weeks as we are switching from env to file config.